geo/variables/GeoVariable.php

```diff
@@ -3,13 +3,13 @@
 
 class GeoVariable
   {
-  public function data()
+  public function data($ip='')
```

Comment:
i think we want to move the default to `fetchIpData`

Reply:
right? unless you are getting an error the other way?

Reply:
I think it needs to stay here. It is throwing an error the other way

Reply:
neat